src/runtime/lpython_builtin.py

```diff
@@ -2,29 +2,22 @@
 #from sys import exit
 
 
-def ord(s: str) -> i32: # currently supports characters with unicode value between 32 to 126
+def ord(s: str) -> i32:
     """
     Returns an integer representing the Unicode code
     point of a given unicode character. This is the inverse of `chr()`.
     """
     if len(s) != 1:
         return -1 # not a character
-    i: i32
-    for i in range(32, 127):
-        if chr(i) == s:
-            return i
+    return i32(int(s))
 
 
-def chr(i: i32) -> str: # currently supports unicode values between 32 to 126
+def chr(i: i32) -> str:
     """
     Returns the string representing a unicode character from
     the given Unicode code point. This is the inverse of `ord()`.
     """
-    if i < 32 or i > 126:
-        return "Not yet supported"
-    all_chars: str
-    all_chars = ' !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~'
-    return all_chars[i - 32]
+    return str(i)
```

Comment:
Do these changes work with CPython also? It seems it does not:
```
In [1]: a = "x"

In [2]: int(a)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[2], line 1
----> 1 int(a)

ValueError: invalid literal for int() with base 10: 'x'
```

Reply:
Both int and str convert decimal digits. But we want their ASCII / unicode values instead. So something is fundamentally wrong here. I think we need to fix LPython for str/int.

Reply:
>Both int and str convert decimal digits. But we want their ASCII / unicode values instead. So something is fundamentally wrong here. I think we need to fix LPython for str/int.

I see. For this PR to proceed further, what should we do?

Reply:
I think we would need to implement ASR nodes or IntrinsicFunctions for chr and ord.

Reply:
Got it. Thank you for the guidance.

Reply:
>I think we would need to implement ASR nodes or IntrinsicFunctions for chr and ord.

Shall we implement the `chr` and `ord` functions as ASR nodes or as IntrinsicFunctions? In case we implement as ASR nodes, do we need to call into `C` or do we implement it using `llvm` instructions?